src/pybind/mgr/volumes/fs/volume.py

```diff
@@ -626,7 +626,7 @@ def list_subvolume_groups(self, **kwargs):
         try:
             with open_volume(self, volname) as fs_handle:
                 volume_exists = True
-                groups = listdir(fs_handle, self.volspec.base_dir)
+                groups = listdir(fs_handle, self.volspec.base_dir, filter_entries=[b"_deleting"])
```

Comment:
better to use `Trash.GROUP_NAME.encode('utf-8')` ?

Reply:
Done